src/Widgets/TopNavigationWidget/SubComponents/LanguageSwitch.tsx

```diff
@@ -37,9 +37,7 @@ export const LanguageSwitch = connect(function LanguageSwitch() {
   return (
     <InPlaceEditingOff>
       <NavDropdown
-        title={
-          <LanguageLabel root={activeSite} className="hidden-md hidden-lg" />
-        }
+        title={<LanguageLabel root={activeSite} className="d-none" />}
```

Comment:
* [ ] An unconditional `d-none` looks suspicious. Can we simplify, e.g., remove?

Reply:
I'm going to rename the `className` prop.